python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_chaos_cleaning_strategy.py

```diff
@@ -0,0 +1,588 @@
+        multiindex = enumerate_function(indices[k])
+        degree = sum(multiindex)
+        c = coefficients[k][0]
+        if abs(c) > threshold:
+            print("#%d, %s (%s) : %s" % (k, multiindex, degree, c))
+            print_index += 1
+        if print_index > maximum_number_of_printed_coefficients:
+            break
+    return
+
+
+# %%
+#
+# The next function computes the polynomial chaos Q2 score using simple validation on a test sample generated by Monte-Carlo sampling. The actual computation is performed by the `MetaModelValidation` class.
+
+# %%
+def compute_polynomial_chaos_Q2(
+    polynomialchaos_result, g_function, input_distribution, n_valid=1000
+):
+    """
+    Compute the Q2 score of the polynomial chaos. 
+    
+
+    Parameters
+    ----------
+    polynomialChaosResult : ot.FunctionalChaosResult
+        The polynomial chaos expansion.
+    g_function : ot.Function
+        The function.
+    input_distribution : ot.Distribution
+        The input distribution.
+    n_valid : int
+        The number of simulations to compute the Q2 score.
+    
+    Returns
+    -------
+    Q2 : float
+        The Q2 score
+    """
+    ot.RandomGenerator.SetSeed(1976)
+    metamodel = polynomialchaos_result.getMetaModel()
+    inputTest = input_distribution.getSample(n_valid)
+    outputTest = g_function(inputTest)
+    val = ot.MetaModelValidation(inputTest, outputTest, metamodel)
+    Q2 = val.computePredictivityFactor()[0]
+    return Q2
+
+
+# %%
+#
+# The following function creates a validation plot using the `draw` method of the `MetaModelValidation` class.
+
+# %%
+def draw_polynomial_chaos_validation(
+    polynomialchaos_result, g_function, input_distribution, n_valid=1000
+):
+    """
+    Validate the polynomial chaos. 
+    
+    Create the validation plot.
+    
+
+    Parameters
+    ----------
+    polynomialChaosResult : ot.FunctionalChaosResult
+        The polynomial chaos expansion.
+    g_function : ot.Function
+        The function.
+    input_distribution : ot.Distribution
+        The input distribution.
+    n_valid : int
+        The number of simulations to compute the Q2 score.
+    
+    Returns
+    -------
+    Q2 : float
+        The Q2 score
+    """
+    metamodel = polynomialchaos_result.getMetaModel()
+    inputTest = input_distribution.getSample(n_valid)
+    outputTest = g_function(inputTest)
+    val = ot.MetaModelValidation(inputTest, outputTest, metamodel)
+    Q2 = val.computePredictivityFactor()[0]
+    graph = val.drawValidation()
+    graph.setTitle("Q2=%.2f%%" % (Q2 * 100))
+    view = otv.View(graph, figure_kw={"figsize": (5.0, 4.0)})
+    return view
+
+
+# %%
+#
+# We consider the Ishigami model which has three inputs uniform in the :math:`[-\pi, \pi]` interval. This is an interesting example for our purpose because it is highly non linear, so that a high polynomial degree will be required in order to produce a polynomial chaos expansion with Q2 score sufficiently close to 1.
```

Comment:
```suggestion
# We consider the Ishigami model which has three inputs that are uniformly distributed on :math:`[-\pi, \pi]` interval.
# This is an interesting example for our purpose because it is highly non linear, so that a high polynomial degree will be required in order to produce a polynomial chaos expansion with Q2 score sufficiently close to 1.
```